pkgs/top-level/perl-packages.nix

```diff
@@ -7727,17 +7727,43 @@ let self = _self // overrides; _self = with self; {
       description = "Embed a Perl interpreter in the Apache HTTP server";
     };
   };
-
   Mojolicious = buildPerlPackage {
-    name = "Mojolicious-4.63";
+    name = "Mojolicious-6.56";
     src = fetchurl {
-      url = mirror://cpan/authors/id/S/SR/SRI/Mojolicious-4.63.tar.gz;
-      sha256 = "f20f77e86fc560dac1c958e765ed64242dcf6343939ed605b45f2bbe2596d5e9";
+      url = mirror://cpan/authors/id/S/SR/SRI/Mojolicious-6.56.tar.gz;
```

Comment:
Looks like you can re-use `name` here, making it easier to keep url and version in sync